### PR TITLE
Add travis.yml file for building on mono

### LINKED ISSUE #22: Set up Travis CI yaml
Official docs http://docs.travis-ci.com/user/languages/csharp/

Some references that are likely outdated (but might prove helpful):

 *  http://stackoverflow.com/a/16751773/18475 
 * http://pseudomuto.com/development/walkthroughs/2013/08/13/continuous-integration-for-net-with-travis-ci-and-xunit/
 * https://danlimerick.wordpress.com/2013/02/03/build-your-open-source-net-project-on-travis-ci/

### PR DESCRIPTION
**Pull request by gep13**
Resolves #22

### CODE REVIEW

**Comment by ferventcoder** [comment]
Close, needs to be `.travis.yml` - note the `.` to start it off.

**Comment by gep13** [comment]
:angry: ah, how did I miss that?!?!

**Comment by gep13** [comment]
Ooo, that's more like it!

image

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: csharp
+script:
```
Comment (ferventcoder) [inline .travis.yml]: In my head I thought it mentioned we needed a `solution: solution-name.sln` as well?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+language: csharp
+script:
+  - ./build.sh
```
Comment (ferventcoder) [inline .travis.yml]: this may need sudo

**Comment by ferventcoder** [comment]
I think we are going to need `nunit-console` installed as well.

```yaml
install:
  - sudo apt-get install mono-devel mono-gmcs nunit-console
```

File: .travis.yml (hunk 1 above)
Comment (gep13) [inline .travis.yml]: Yeah, I saw that as well, but then I read:

`When the solution key is present, Travis will run NuGet package restore and build the given solution. You can also specify your own scripts, as shown in the next section.`

Which isn't required, i.e. the build process will take care of the building

**Pull request merged**